0.7.0 not building with ocaml 4.03.0

```bash
#=== ERROR while installing protocol-9p.0.7.0 =================================#
# opam-version 1.2.2
# os           linux
# command      make
# path         /home/philip/.opam/4.03.0/build/protocol-9p.0.7.0
# compiler     4.03.0
# exit-code    2
# env-file     /home/philip/.opam/4.03.0/build/protocol-9p.0.7.0/protocol-9p-6802-1872ec.env
# stdout-file  /home/philip/.opam/4.03.0/build/protocol-9p.0.7.0/protocol-9p-6802-1872ec.out
# stderr-file  /home/philip/.opam/4.03.0/build/protocol-9p.0.7.0/protocol-9p-6802-1872ec.err
### stdout ###
# [...]
# ocamlfind ocamldep -package astring -package named-pipe.lwt -package stringext -package lwt -package cstruct.lwt -package result -package cstruct -package fmt -package win-error -package 'cmdliner, logs.fmt' -package lambda-term -package logs -modules src/main.ml > src/main.ml.depends
# ocamlfind ocamlc -c -g -bin-annot -safe-string -package astring -package named-pipe.lwt -package stringext -package lwt -package cstruct.lwt -package result -package cstruct -package fmt -package win-error -package 'cmdliner, logs.fmt' -package lambda-term -package logs -w @5@8@10@11@12@14@23@24@26@29@40 -I src -I lib -I unix -I lib_test -o src/main.cmo src/main.ml
# ocamlfind ocamlopt -c -g -bin-annot -safe-string -package astring -package named-pipe.lwt -package stringext -package lwt -package cstruct.lwt -package result -package cstruct -package fmt -package win-error -package 'cmdliner, logs.fmt' -package lambda-term -package logs -w @5@8@10@11@12@14@23@24@26@29@40 -I src -I lib -I unix -I lib_test -o src/main.cmx src/main.ml
# ocamlfind ocamlopt -linkpkg -g -package astring -package named-pipe.lwt -package stringext -package lwt -package cstruct.lwt -package result -package cstruct -package fmt -package win-error -package 'cmdliner, logs.fmt' -package lambda-term -package logs lib/protocol_9p_error.cmx lib/protocol_9p_s.cmx lib/protocol_9p_buffered9PReader.cmx lib/protocol_9p_infix.cmx lib/protocol_9p_types.cmx lib/protocol_9p_request.cmx lib/protocol_9p_response.cmx lib/protocol_9p_client.cmx lib/protocol_9p_info.cmx lib/protocol_9p_filesystem.cmx lib/protocol_9p_server.cmx lib/protocol_9p.cmx unix/flow_lwt_unix.cmx unix/client9p_unix.cmx unix/lofs9p.cmx unix/server9p_unix.cmx src/main.cmx -o src/main.native
# + ocamlfind ocamlopt -linkpkg -g -package astring -package named-pipe.lwt -package stringext -package lwt -package cstruct.lwt -package result -package cstruct -package fmt -package win-error -package 'cmdliner, logs.fmt' -package lambda-term -package logs lib/protocol_9p_error.cmx lib/protocol_9p_s.cmx lib/protocol_9p_buffered9PReader.cmx lib/protocol_9p_infix.cmx lib/protocol_9p_types.cmx lib/protocol_9p_request.cmx lib/protocol_9p_response.cmx lib/protocol_9p_client.cmx lib/protocol_9p_info.cmx lib/protocol_9p_filesystem.cmx lib/protocol_9p_server.cmx lib/protocol_9p.cmx unix/flow_lwt_unix.cmx unix/client9p_unix.cmx unix/lofs9p.cmx unix/server9p_unix.cmx src/main.cmx -o src/main.native
# File "_none_", line 1:
# Error: No implementations provided for the following modules:
#          Lwt_unix referenced from /home/philip/.opam/4.03.0/lib/named-pipe/named-pipe_lwt.cmxa(Named_pipe_lwt)
#          Channel referenced from lib/protocol_9p_buffered9PReader.cmx
# Command exited with code 2.
### stderr ###
# make: *** [Makefile:41: build] Error 10
```

--- a/src/_tags
+++ b/src/_tags
@@ -4,3 +4,6 @@
 <*.*>: package(cmdliner, logs.fmt)
 <*.*>: package(astring), package(named-pipe.lwt)
 <*.*>: package(win-error)
+<*.*>: package(channel)
+<*.*>: package(io-page.unix)
+<*.*>: package(lwt.unix)

